src/labtools/adtools/counter.py

```diff
@@ -31,17 +31,24 @@ def seq_counter(fastq, design_to_use = None, barcoded = False, only_bcs = False,
     dtype: int64
     """
     seqCounts = {}
+    # LT: Extract loss_table from kwargs and make design_loss_table to hold temporary loss_table calculations
+    loss_table = kwargs.pop('loss_table', None) 
+    design_loss_table = {'total': 0, 'filtered': 0}
```

Comment:
I think you should either add loss_table as a required argument to this function or handle the event of loss table not being in kwargs. Right now, if loss_table is None then you will get `TypeError: 'NoneType' object does not support item assignment` in `pull_barcode` or `pull_AD` since those functions don't handle the loss_table being None.